packages/realm_dart/lib/src/configuration.dart

```diff
@@ -187,19 +187,18 @@ abstract class Configuration {
     int? maxNumberOfActiveVersions,
     ShouldCompactCallback? shouldCompactCallback,
     int schemaVersion = 0,
+    bool cancelAsyncOperationsOnNonFatalErrors = false,
   }) =>
-      FlexibleSyncConfiguration._(
-        user,
-        schemaObjects,
-        fifoFilesFallbackPath: fifoFilesFallbackPath,
-        path: path,
-        encryptionKey: encryptionKey,
-        syncErrorHandler: syncErrorHandler,
-        clientResetHandler: clientResetHandler,
-        maxNumberOfActiveVersions: maxNumberOfActiveVersions,
-        shouldCompactCallback: shouldCompactCallback,
-        schemaVersion: schemaVersion,
-      );
+      FlexibleSyncConfiguration._(user, schemaObjects,
+          fifoFilesFallbackPath: fifoFilesFallbackPath,
+          path: path,
+          encryptionKey: encryptionKey,
+          syncErrorHandler: syncErrorHandler,
+          clientResetHandler: clientResetHandler,
+          maxNumberOfActiveVersions: maxNumberOfActiveVersions,
+          shouldCompactCallback: shouldCompactCallback,
+          schemaVersion: schemaVersion,
+          cancelAsyncOperationsOnNonFatalErrors: cancelAsyncOperationsOnNonFatalErrors);
```

Comment:
nitpick for better formatting..
```suggestion
          cancelAsyncOperationsOnNonFatalErrors: cancelAsyncOperationsOnNonFatalErrors,);
```